contrib/ros/rikai/contrib/ros/__init__.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright (c) 2021 Rikai Authors
```

Comment:
Are we moving the other contrib modules out as well? What should go in /contrib vs /python/rikai/contrib?

Reply:
yea, these contrib module `namespace` based packaing, we should put them under `/contrib/<libname>`. 

After installation, they can be universally accessed via `rikai.contrib.<lilbname>...`.